### PR TITLE
[cleanup] Remove dead code in several React components

### PR DESCRIPTION
**Pull request by vincentfretin**
Those are props and state that aren't used anywhere.

### CODE REVIEW

File: src/components/scenegraph/SceneGraph.js [hunk 1]
```diff
@@ -70,19 +66,16 @@ export default class SceneGraph extends React.Component {
...
       const entityOption = this.state.filteredEntities[i];
       if (entityOption.entity === entity) {
-        this.setState({ selectedEntity: entity, selectedIndex: i });
+        this.setState({ selectedIndex: i });
         // Make sure selected value is visible in scenegraph
         this.expandToRoot(entity);
-        if (this.props.onChange) {
-          this.props.onChange(entity);
-        }
-        Events.emit('entityselect', entity, true);
```
Comment (vincentfretin) [inline src/components/scenegraph/SceneGraph.js:84]: This true param here has no meaning, there is no second parameter in any of the entityselect listener in the code base, meaning there is no `Events.on('entityselect', (entity, someboolean) => ...`

**Comment by dmarcos** [comment]
Thanks!

**Comment by vincentfretin** [comment]
I rebased to fix the conflict. You can merge.

**Comment by vincentfretin** [comment]
@dmarcos in case you missed my comment.

**Comment by dmarcos** [comment]
Thanks!

**Pull request merged**